Game/ConsoleApplication2/MovingGameObject.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "GameObject.h"
+#include "Constants.h"
+
+
+class MovingGameObject : public GameObject
+{
+public:
+	// With acceleration
+	MovingGameObject(Rect bounds, Vector2D acceleration, Vector2D maxSpeed, bool isMiniMapObject = false)
+		: GameObject(bounds),
+		m_ACCELERATION(acceleration),
```

Comment:
member consts do not need 'm_' it is assumed unless otherwise in another namespace, this is because un-scoped global vars are against the coding standard. But again i'm not gonna be too picky about this project, just telling you for the future